sympy/physics/control/lti.py

```diff
@@ -1134,6 +1138,8 @@ def __rsub__(self, other):
         return -self + other
 
     def __mul__(self, other):
+        if(isinstance(other, Feedback)):
```

Comment:
why `if(isinstance(other, Feedback)):` and not just if `isinstance(other, Feedback):`  as you did before?

Reply:
Thankyou for pointing it out. I've corrected it .